### PR TITLE
v1.37.0 Update api-updates.rss

### PR DESCRIPTION
**Pull request by lengocduong1**

### CODE REVIEW

File: feed/api-updates.rss [hunk 1]
```diff
@@ -5,6 +5,13 @@
     <link>https://docs.upvest.co/news</link>
     <description>Updates of the API release notes</description>
     <pubDate>Thu, 26 Sep 2024 13:00:00 +0000</pubDate>
```
Comment (upmkrasiukov) [inline feed/api-updates.rss:7]: needs to be updated too

File: feed/api-updates.rss [hunk 2]
```diff
@@ -5,6 +5,13 @@
...
+    <item>
+    <title>v1.37.0 Corporate action notifications</title>
```
Comment (upmkrasiukov) [inline feed/api-updates.rss:9]: need +1 indentation

File: feed/api-updates.rss [hunk 3]
```diff
@@ -4,7 +4,14 @@
     <title>Upvest API updates</title>
     <link>https://docs.upvest.co/news</link>
     <description>Updates of the API release notes</description>
-    <pubDate>Thu, 26 Sep 2024 13:00:00 +0000</pubDate>
+    <pubDate>Tue, 01 Oct 2024 13:00:00 +0000</pubDate>
+    <item>
+      <title>v1.37.0 Corporate action notifications</title>
+      <description>Operation models: TOL, BYOL, OMNIBUS\\nChanges: ...</description>
```
Comment (upmkrasiukov) [inline feed/api-updates.rss:10]: Would you like to write a short summary of the changes?
Also for the new line use `&lt;br/&gt;` (updated the guide)

**Review by upmkrasiukov — CHANGES_REQUESTED** [review]

File: feed/api-updates.rss [hunk 4]
```diff
@@ -4,7 +4,14 @@
     <title>Upvest API updates</title>
     <link>https://docs.upvest.co/news</link>
     <description>Updates of the API release notes</description>
-    <pubDate>Thu, 26 Sep 2024 13:00:00 +0000</pubDate>
+    <pubDate>Tue, 01 Oct 2024 13:00:00 +0000</pubDate>
```
Comment (upmkrasiukov) [inline feed/api-updates.rss:7]: should be similar to item's pubDate to pass validation

**Review by forik — DISMISSED** [review]

**Review by upmkrasiukov — APPROVED** [review]

**Pull request merged**